Beecrowd.Beginner.Solutions/1036-BashkarasFormula.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Beecrowd
+{
+    class BhaskarasFormula
+    {
+        public static void Run()
+        {
+            string[] inputs = Console.ReadLine().Split(' ');
+            double a = Convert.ToDouble(inputs[0]);
+            double b = Convert.ToDouble(inputs[1]);
+            double c = Convert.ToDouble(inputs[2]);
+
+            if (a == 0)
+            {
+                Console.WriteLine("Impossivel calcular");
```

Comment:
Put this string in a const read-only string variable and used it in both places.

Reply:
Yep, you are right, it can be const.